action.yml

```
@@ -157,6 +182,34 @@ runs:
         fi
         rm "$FILELIST"
         exit $RETVAL
+
+    - name: Run PHP Mess Detector checks
+      shell: bash
+      run: |
+        FILELIST=$(mktemp /tmp/phpmd-check.XXXXXX)
+        mapfile -d ',' -t added_modified_files < <(printf '%s,' '${{ steps.files.outputs.added_modified }}')
+        for FILE in "${added_modified_files[@]}"; do
+          EXT="${FILE##*.}"
+          echo "Changed file: " $FILE ", extension: " $EXT
+          if [[ "$EXT" == "php" && $FILE != migrations/Version* ]]; then
+            echo "src/${FILE}" >> "${FILELIST}"
+            echo "File added to the file list: " $FILE
+          fi
```

I think Migrations can be included on this check 

```suggestion
          if [[ "$EXT" == "php" ]]; then
            echo "src/${FILE}" >> "${FILELIST}"
            echo "File added to the file list: " $FILE
          fi
```

---

Do you think we should also enable the PSR-12 check for migration files as part of this PR? Currently it has the same condition to skip migrations (`if [[ "$EXT" == "php" && $FILE != migrations/Version* ]];`)

---

PSR was blocked because the migration class namespace is not supported on PSR-12, I did not agree with that, but lets not increase the scope of this PR, please :)